demosplan/DemosPlanCoreBundle/Resources/config/services.yml

```diff
@@ -31,6 +31,193 @@ services:
+
+    DemosEurope\DemosplanAddon\Contracts\Events\ParameterProviderEventInterface:
+        class: DemosEurope\DemosplanAddon\DemosPipes\Event\ParameterProviderEvent
+
+    DemosEurope\DemosplanAddon\Contracts\Events\DailyMaintenanceEventInterface:
+        class: demosplan\DemosPlanCoreBundle\Event\DailyMaintenanceEvent
+
+    DemosEurope\DemosplanAddon\Contracts\Events\AfterResourceCreationEventInterface:
+        class: demosplan\DemosPlanCoreBundle\Event\AfterResourceCreationEvent
+
+    DemosEurope\DemosplanAddon\Contracts\Events\IsFileDirectlyAccessibleEventInterface:
+        class: demosplan\DemosPlanCoreBundle\Event\IsFileDirectlyAccessibleEvent
+
+    DemosEurope\DemosplanAddon\Contracts\Events\AfterResourceUpdateEventInterface:
+        class: demosplan\DemosPlanCoreBundle\Event\AfterResourceUpdateEvent
+
+    DemosEurope\DemosplanAddon\Contracts\Events\IsFileAvailableEventInterface:
+        class: demosplan\DemosPlanCoreBundle\Event\IsFileAvailableEvent
+
+    DemosEurope\DemosplanAddon\Contracts\Services\StatementServiceInterface:
+        class: demosplan\DemosPlanStatementBundle\Logic\StatementService
+
+    DemosEurope\DemosplanAddon\Contracts\Services\CurrentUserProviderInterface:
+        class: demosplan\DemosPlanUserBundle\Logic\CurrentUserService
+
+    DemosEurope\DemosplanAddon\Contracts\Services\ProcedureServiceInterface:
+        class: demosplan\DemosPlanProcedureBundle\Logic\ProcedureService
+
+    DemosEurope\DemosplanAddon\Contracts\Entities\FileInterface:
+        class: demosplan\DemosPlanCoreBundle\Entity\File
+
+    DemosEurope\DemosplanAddon\Contracts\Entities\StatementInterface:
+        class: demosplan\DemosPlanCoreBundle\Entity\Statement\Statement
+
+    DemosEurope\DemosplanAddon\Contracts\Entities\ProcedureInterface:
+        class: demosplan\DemosPlanCoreBundle\Entity\Procedure\Procedure
+
+    DemosEurope\DemosplanAddon\Contracts\Entities\SegmentInterface:
+        class: demosplan\DemosPlanCoreBundle\Entity\Statement\Segment
+
+    DemosEurope\DemosplanAddon\Contracts\Entities\CoreEntityInterface:
+        class: demosplan\DemosPlanCoreBundle\Entity\CoreEntity
+
+    DemosEurope\DemosplanAddon\Contracts\Entities\TagInterface:
+        class: demosplan\DemosPlanCoreBundle\Entity\Statement\Tag
+
+#    DemosEurope\DemosplanAddon\Contracts\Entities\UuidEntityInterface:
+#        class:
+
+#    DemosEurope\DemosplanAddon\Contracts\Entities\CustomerInterface:
+#        class:
+
+#    DemosEurope\DemosplanAddon\Contracts\Entities\EntityChangeInterface:
+#        class:
+
+#    DemosEurope\DemosplanAddon\Contracts\Entities\EntityInterface:
+#        class:
+
+    DemosEurope\DemosplanAddon\Logic\Rpc\RpcErrorGeneratorInterface:
+        class: demosplan\DemosPlanCoreBundle\Logic\Rpc\RpcErrorGenerator
+
+    DemosEurope\DemosplanAddon\Permission\Validation\PermissionFilterValidatorInterface:
+        class: demosplan\DemosPlanCoreBundle\Permissions\PermissionResolver
+
```

Comment:
Is this really necessary? And if so, shouldn't it be in the services.yml of the addon-repo?

Reply:
If you have a brilliant idea on how to bind services in the addon repo which the addon repo doesn't and mustn't know about, I'm all ears.